#RecordRequest.srv
float32 duration
---
string response
